app/lib/pre_assembly/from_file_manifest/file_set_builder.rb

```diff
@@ -25,7 +25,7 @@ def initialize(resource:, cocina_dro:, external_identifier:, staging_location:)
 
       def build
         {
-          label: resource[:label],
+          label: resource[:label] || '',
```

Comment:
Is it worth adding a comment here about why we are setting to the empty string?

Reply:
I thought about this when writing, but couldn't come up with a useful comment that said more that wasn't clear from the code.